pages/common/docker-cp.md

```diff
@@ -0,0 +1,12 @@
+# docker cp
+
+> Copy files/folders between a container and the host filesystem.
```

Comment:
It doesn't need to match the original wording at all.  You can make it whatever you want, as long as it's consise and gets the point across (although it may be reworded :)